BoxContentSDK/BoxContentSDK/BOXURLSessionCacheClient.m

```diff
@@ -139,6 +139,34 @@ - (BOOL)cacheUserId:(NSString *)userId
     return success;
 }
 
+- (BOOL)cacheSessionTaskStartedGivenUserId:(NSString *)userId
+                               associateId:(NSString *)associateId
+                                     error:(NSError **)error
+{
+    if (userId == nil || associateId == nil) {
+        if (error != nil) {
+            *error = [[NSError alloc] initWithDomain:BOXContentSDKErrorDomain
```

Comment:
I guess since we have the error code already it's not too bad, but it strikes me that this error should be compile checked or an assertion, since it's simple API misuse.

Reply:
agree, will add comment to address this with other similar usage as well